<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(126, 126, 126);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="11">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Segmentation Training Tool</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnLoadDataset">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Dataset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="btnOpenmodel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="btnTestModel">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Model</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnLoadClasses">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Classes.txt</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max Epochs :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="edtMaxepochs">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Batch Size :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QLineEdit" name="edtBatchsize">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Input Size :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" colspan="2">
     <widget class="QLineEdit" name="edtInputsize">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QPushButton" name="btnExportC">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Export For C++</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>517</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="6" colspan="4">
     <widget class="QPushButton" name="btnCheckDB">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Check Dataset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10">
     <widget class="QPushButton" name="btnStartTrain">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Train</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="11">
     <widget class="QListWidget" name="listStatus"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Initial Weights :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="7">
     <widget class="QLineEdit" name="edtInitialWeights">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
